bindings/pyroot_experimental/PyROOT/src/PyROOTStrings.h

```diff
@@ -0,0 +1,28 @@
+
+#ifndef PYROOT_PYSTRINGS_H
+#define PYROOT_PYSTRINGS_H
+
+#include "Python.h"
+#include "DllImport.h"
+
+namespace PyROOT {
+
+// python strings kept for performance reasons
+
+namespace PyStrings {
+
+R__EXTERN PyObject *gBranch;
```

Comment:
Could we instead of these globals have a function for each, with a function-local static, that initializes the static upon first invocation? That avoids init issues (use before init, threading issues, init dependency issues)

Reply:
Yes we could do that, just want to point out that it would modify the behaviour of the extension module in a couple of ways:
- Now these Python strings are created during the initialization of the extension module. If the creation of one of them fails, the module fails to initialize and we get an early error. Making their creation lazy would delay that error until some code uses them.
- The Python strings are now properly cleaned (`DECREF`ed) on Python interpreter shutdown. We would not have this `DECREF` with function-local statics, but we would be closing down the interpreter anyway...

Reply:
OK then these `nullptr`s are actually useful, to detect a missing module initialization / use of the module before its initialization. Good, please ignore my comment.